index.html

```diff
@@ -13,58 +13,57 @@
   <div class="container">
     <h1>Shopping List</h1>
 
-    <form id="js-shopping-list-form">
-      <label for="shopping-list-entry">Add an item</label>
+    <form id="js-shopping-list-add-item">
```

Comment:
The `js-` prefix is reserved for ids and classes that are to be targeted by JS layer. Shouldn't be used for CSS. The one that's in master right now should actually be gone altogether, so if you need an id here for CSS, can create new one.